packages/forklift-console-plugin/src/modules/Providers/views/create/components/EsxiProviderCreateForm.tsx

```diff
@@ -87,9 +81,9 @@ export const EsxiProviderCreateForm: React.FC<EsxiProviderCreateFormProps> = ({
         onChange({
           ...provider,
           spec: {
+            ...provider?.spec,
             type: provider.spec.type,
             url: provider.spec.url,
-            ...provider?.spec,
```

Comment:
I am aware of that  but I didn't like the original order since if we would like to change it in the future and overwrite the value, it won't behave as expected.  I'm not sure that adding those 2 lines contribute to code clarification....therefore it's really better to remove those 2 lines in relevant places:
```
 type: provider.spec.type,
 url: provider.spec.url,
```